tutorials/abap-environment-create-rap-business-events/abap-environment-create-rap-business-events.md

```diff
@@ -0,0 +1,543 @@
+---
```

Comment:
In many places in this tutorial, a specific instruction starts with "You need..."  Instead of the phrasing, please start with an action verb: Create, Save, Open, etc.  Many lines do start with this, but some start with "You need..."  Please change this so the instructions are more clear and direct.